wold2/locale/en/LC_MESSAGES/clld.po

```diff
@@ -1,46 +1,48 @@
 # English translations for wold2.
-# Copyright (C) 2013 ORGANIZATION
+# Copyright (C) 2018 ORGANIZATION
 # This file is distributed under the same license as the wold2 project.
-# FIRST AUTHOR <EMAIL@ADDRESS>, 2013.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2018.
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: wold2 0.0\n"
 "Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
-"POT-Creation-Date: 2013-10-30 13:06+0100\n"
-"PO-Revision-Date: 2013-10-22 10:46+0200\n"
+"POT-Creation-Date: 2018-03-07 13:34+0100\n"
+"PO-Revision-Date: 2018-03-07 13:34+0100\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: en\n"
 "Language-Team: en <LL@li.org>\n"
 "Plural-Forms: nplurals=2; plural=(n != 1)\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Generated-By: Babel 1.3\n"
+"Generated-By: Babel 2.5.3\n"
 
-#: wold2/__init__.py:12
+#: wold2/__init__.py:15
 msgid "Contribution"
-msgstr "Vocabulary"
+msgstr ""
```

Comment:
So it seems as if this file has been recreated, thereby losing the existing translations - possibly due to renaming the catalog file?